OSDOCS-14169:CUDN localnet (#93001)
openshift/openshift-docs: modules/support-matrix-for-udn-nad.adoc

[NOTE]
====

> As of {product-title} 4.19, the Localnet topology is generally available for the `ClusterUserDefinedNetwork` CRs and is the preferred method for connecting your physical network to your virtual one. Alternatively, the `NetworkAttachmentDefinition` CR can leverage secondary networks to create Localnet topologies as well.

Reviewer: ```suggestion
As of {product-title} 4.19, the `Localnet` topology is generally available for the `ClusterUserDefinedNetwork` CRs and is the preferred method for connecting physical networks to virtual networks. Alternatively, the `NetworkAttachmentDefinition` CR can also be used to create `Localnet` topologies with secondary networks.
```
